### PR TITLE
Improve RSOD experience

### LINKED ISSUE #32: Define standard for error message generation
Typically, the RSODs and AliceOS errors in generally follow the same method of naming errors as Microsoft does with BSODs. However, it can get difficult trying to name it in this fashion. We should define a way to write errors that are human readable/understandable and are unique to AliceOS.

### PR DESCRIPTION
**Pull request by alicerunsonfedora**
screenshot0046

## Proposed changes
- QR code displayed on top right to quickly access the Error Database (https://errordb.aliceos.app)
- Text changed to "If you'd like to know more, you can search the error database later for this error"
- Background updated to dead Alice on floor (per Noelia's suggestion)
- RSOD text now rests in `vbox` with `null height`s separating them
- Error messages logged and documented in Error Database

### If approved, will do the following:
- Close #32

### CODE REVIEW

File: CoreServices/ASErrorHandler.rpy [hunk 1]
```diff
@@ -4,40 +4,77 @@
...
+        }
+
+        siperr = {
+            "SIPComponentFailure": "SIP_FAILED_INIT",
+            "SIPCriticalFileCheckFail": "SIP_CRITICAL_FILE_INVALID",
+            "SIPComponentProfileFail": "SIP_COMPONENT_FAILED"
+        }
+
+        pisaerr = {
+            "NullInput": "INPUT_REUTRNED_NULL"
```
Comment (sr229) [inline CoreServices/ASErrorHandler.rpy]: Change this to `NULL_IN_NON_NULLABLE_CTX`

**Review by sr229 — CHANGES_REQUESTED** [review]

File: CoreServices/ASErrorHandler.rpy (hunk 1 above)
Comment (alicerunsonfedora) [reply CoreServices/ASErrorHandler.rpy]: That is human-readable, but I don't think it's quite human-_understandable_...

Comment (sr229) [reply CoreServices/ASErrorHandler.rpy]: It's for programmer purposes so I don't think a end-user would need to learn about that.

**Comment by alicerunsonfedora** [comment]
@sr229 Requesting review updates:
* Updated `NullInput` string value to `NULL_IN_NON_NULL_CTX`

**Comment by sr229** [comment]
QR Code should be white against dark red.

**Comment by alicerunsonfedora** [comment]
I'll try to invert the colors again and see what happens. I ran into an issue last time where it looked fine, but the QR code itself failed to load correctly.

**Review by sr229 — APPROVED** [review]
LGTM

**Comment by sr229** [comment]
Since @Gyrosona is no longer active, my approval takes precednece, merging

**Pull request merged**